content/2_Bronze/Complete_Rec.mdx

```diff
@@ -122,8 +122,10 @@ public class Main {
 			return Math.abs(s1_sum - s2_sum);
 		}
 
-		return Math.min(solve(index + 1, s1_sum + weights[index], s2_sum),
-						solve(index + 1, s1_sum, s2_sum + weights[index]));
+		return Math.min(
```

Comment:
the formatting looks a bit weird- did you use spaces or something?

Reply:
looks like this: 
![image](https://user-images.githubusercontent.com/63619830/120899885-ba14b000-c5e6-11eb-9ad3-7c984d82856c.png)
is that okay?